app/views/hyku_addons/dates/_date_fields.html.erb

```diff
@@ -6,7 +6,7 @@
 
 		<% # The form-group class "#{template}_date_#{date_type}_..." is used for DOI auto population %>
 		<div class="form-group <%= "#{template}_date_#{date_type}_year" %>">
-			<%= select_year(f.object.model.send("date_#{date_type}").to_s.split('-')[0].to_i, {start_year: (Date.today.year + 6), end_year: (Date.today.year - 159), prompt: "Select year"}, {id: "#{template}_date_#{date_type}__date_#{date_type}_year", name: "#{template}[date_#{date_type}][][date_#{date_type}_year]", class: "form-control ubiquity-date-#{date_type}-year ubiquity-date-input #{template}_date_#{date_type}_year" }) %>
+			<%= select_year(f.object.model.send("date_#{date_type}").to_s.split('-')[0].to_i, {start_year: (Date.today.year + 6), end_year: 1750, prompt: "Select year"}, {id: "#{template}_date_#{date_type}__date_#{date_type}_year", name: "#{template}[date_#{date_type}][][date_#{date_type}_year]", class: "form-control ubiquity-date-#{date_type}-year ubiquity-date-input #{template}_date_#{date_type}_year" }) %>
```

Comment:
Are the start and end year options switched? Reading the code gives that impression

Reply:
I believe the start year is the top of the list, so we want to present more recent years first in the dropdown